src/sklearn_evaluation/util.py

```diff
@@ -39,14 +40,37 @@ def _can_iterate(obj):
     return is_iterable and not is_string
 
 
+def check_elements_in_range(array, min, max, include_min=True, inclue_max=True):
```

Comment:
It can be renamed to `get_elements_in_range`

Reply:
I think `get_elements_in_range` implies a different case.
This method checks if values in an array are within a range. I can change it to something like `check_array_range`. what do you think? @neelasha23

Reply:
ok I got it now!